docfx.json

```
@@ -26,7 +26,7 @@
         ],
         "src": "non-version"
       },
-      { "files": [ "**/*.md", "**/*.yml" ], "group": "webview2-1072-54", "src": "1-0-1072-54" },
+      { "files": [ "**/*.md", "**/*.yml" ], "group": "webview2-1108-44", "src": "1-0-1108-44" },
```

We'd want to keep both entries.

```suggestion
      { "files": [ "**/*.md", "**/*.yml" ], "group": "webview2-1108-44", "src": "1-0-1108-44" },
      { "files": [ "**/*.md", "**/*.yml" ], "group": "webview2-1072-54", "src": "1-0-1072-54" },
```